components/Badge/Badge.css

```diff
@@ -0,0 +1,50 @@
+/* 
+    Author Name:Nikhil Nirmal,
+    color pallet used:
+    background:#0a192f;
+    font 1 : #55dabe;
+    font 2 : #ccd6f6;
+    font 3 : #6e7996;   
+*/
+
+.badge{
+    position: relative;
+    width: max-content;
+}
+
+.badge > div{
+    bottom: 8px;
+    right: 6px;
+    position: absolute;
+    height: 10px;
+    width: 10px;
+    border-radius: 50%;
+    border: 1px solid rgb(255, 255, 255);
+}
+
+.on {
+    background-color: greenyellow;
+}
+
+.off {
+    background-color: #757575;
+}
+
+.dnd {
+    background-color: red;
+}
+.badge .num {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    top : -.2rem;
+    right: -.1rem;
+    position: absolute;
+    font-size: .8rem;
+    color: #ffffff;
+    background-color: #5cc3ff;
```

Comment:
why not using color variables everywhere?

Reply:
Updated with color variables.